internal/statemachine/helper.go

```diff
@@ -867,5 +795,5 @@ func associateLoopDevice(path string, sectorSize quantity.Size) (string, *exec.C
 // divertOSProber divert GRUB's os-prober as we don't want to scan for other OSes on
 // the build system
 func divertOSProber(mountDir string) (*exec.Cmd, *exec.Cmd) {
-	return dpkgDivert(mountDir, "/etc/grub.d/30_os-prober")
+	return helperDpkgDivert(mountDir, "/etc/grub.d/30_os-prober")
```

Comment:
nipick: No need to fix it now, but at some point out of consistency `divertOSProber` should probably also live in the `helper` package. But this PR is already much bigger than anticipated so let's live with that for now :).